### DISCUSSION TITLE
Next 13 not show not found page when routing not found

### DISCUSSION THREAD

**Discussion by phatify** [vercel/next.js#42276] [Help]
When I created new routing:  `/pages/user/[id].js`, then I trying access this route from URL (`http://localhost:3000/user`) without` [id].js` and got a error message: `Application error: a client-side exception has occurred (see the browser console for more information).`. It never show error not found page. Why? I think this is a bug and not the feature!

**Comment by erikbg7** [comment]
Could you provide more info about your folder architecture or the error shown in the console? Regards
